haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/model/House.java

```diff
@@ -1,4 +1,82 @@
 package cz.muni.fi.pa165.hauntedhouses.model;
 
+
+import javax.persistence.*;
+import java.util.Objects;
+
+/**
+ * @author Petr Vitovsky
+ */
+
+@Entity
 public class House {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false, unique = true)
+    private String address;
+
+    @Column
+    private java.time.LocalDate hauntedSince;
+
+    @Column
+    private String history;
+
+    public long getId() {
+        return this.id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAddress() {
+        return this.address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public String getHistory() {
+        return this.history;
+    }
+
+    public void setHistory(String history) {
+        this.history = history;
+    }
+
+    public java.time.LocalDate getHauntedSince() {
```

Comment:
v kode nema co robit takyto skaredy nazov... pridaj import a nech je tu pekne napisane LocalDate

Reply:
Opraveno

Reply:
Ten navratovy typ je stale java.time.LocalDate. Mozes to prepisat na LocalDate? To iste aj pre:
public void setHauntedSince(java.time.LocalDate hauntedSince) {...}
o par riadkov nizsie

Reply:
Na metody jsem zapomněl, už je to správně.